src/net/p2p.rs

```diff
@@ -3381,7 +3425,10 @@ impl PeerNetwork {
                 get_epoch_time_secs() + self.connection_opts.mempool_sync_timeout;
         } else {
             if get_epoch_time_secs() > self.mempool_sync_timeout {
```

Comment:
What's the different between `self.mempool_sync_timeout` and `self.mempool_sync_deadline`?

Reply:
`self.connection_opts.mempool_sync_timeout` is the maximum amount of time a mempool sync is allowed to take before the peer gives up.  `self.mempool_sync_deadline` is the wall-clock time when the current mempool sync must terminate because it's taking too long.

Reply:
But `self.mempool_sync_timeout` is checked directly against the epoch time here -- so it must also be a deadline, right?

Reply:
Ah, I misread what you were asking.  `self.mempool_sync_timeout` is the absolute time at which the mempool sync will be terminated if it takes too long.  `self.mempool_sync_deadline` is the time at which the _next_ mempool sync will start.  So, once a mempool sync terminates, at least `self.mempool_sync_deadline` seconds will pass until the next mempool sync starts up.

Reply:
That makes sense, thanks -- can you add some rustdocs to those fields in the `PeerNetwork` struct that convey that?